c7n/actions/webhook.py

```diff
@@ -129,6 +128,18 @@ def _process_call(self, resource):
         except urllib3.exceptions.HTTPError as e:
             self.log.error("Error calling %s. Code: %s" % (prepared_url, e.reason))
 
+    def _build_http_manager(self):
+        pool_kwargs = {
+            'cert_reqs': 'CERT_REQUIRED',
+            'ca_certs': certifi and certifi.where() or None
```

Comment:
its not new in this pr, but we should probably support a CA bundle env var here, its come up in the past for some enterprise environments where the endpoint is otherwise not valid, some existing common ones that we could reuse  REQUESTS_CA_BUNDLE, CURL_CA_BUNDLE, SSL_CERT_FILE

Reply:
we can split that out to separate issue for followup